testing/human-pose-estimation/app.py

```diff
@@ -0,0 +1,87 @@
+import logging
+from typing import Tuple, List
+
+import depthai as dai
+import cv2
+import numpy as np
+import blobconverter
+from handler import decode, ImgLandmarks, ALL_COLORS
+
+
+class PoseEstimator:
```

Comment:
I agree. I was trying to maintain the structure from previous examples, but since the class is mainly functional and holds only one class var, it can be removed.